src/zac/core/forms.py

```diff
@@ -323,37 +323,95 @@ class SelectUsersReviewRequestForm(forms.Form):
         help_text=_("Select the advisors."),
     )
 
+    deadline = forms.DateField(
+        required=True,
+        label=_("Deadline"),
+        help_text=_("Select a date"),
+        input_formats=["%Y-%m-%d"],
+    )
+
 
 class BaseReviewRequestFormSet(BaseTaskFormSet):
-    def is_valid(self):
-        super().is_valid()
+    def deadlines_validation(self) -> bool:
+        """
+        Validate that deadlines per step are monotonic increasing
+        """
+        valid = True
+        deadline_old = date(1, 1, 1)
         for form in self.forms:
-            if not form.has_changed():
-                form.add_error(None, _("Please select at least 1 advisor."))
-                return False
+            deadline_new = form.cleaned_data["deadline"]
+            if deadline_new and not deadline_new > deadline_old:
+                form.add_error(
+                    None,
+                    _(
+                        f"Please select a date greater than {deadline_old.strftime('%Y-%m-%d')}"
+                    ),
```

Comment:
This does not work with translatable strings, as the 'template' contains a dynamic value. Please use placeholders - in general, f-strings are not suitable for translatable content.

```suggestion
                    _("Please select a date greater than {minimum_date}").format(minimum_date=deadline_old.strftime('%Y-%m-%d'))
```